deploy/helm/templates/daemonset.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cadvisor
+  namespace: {{.Release.Namespace}}
+  labels:
+    app.kubernetes.io/name: cAdvisor
+    app.kubernetes.io/managed_by: Helm
+spec:
+  selector:
+    matchLabels:
+      name: cadvisor
+  template:
+    metadata:
+      labels:
+        name: cadvisor
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: runtime/default
+    spec:
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      serviceAccountName: cadvisor
+      hostPID: true
+      containers:
+      - name: cadvisor
+        image: {{.Values.image}}
+        imagePullPolicy: {{.Values.imagePullPolicy}}
+        args:
+          {{if .Values.perf}}- --perf_events_config=/etc/config/perf-config.json{{end}}
+          {{if .Values.disable_metrics}}- --disable_metrics={{ join "," .Values.disable_metrics}}{{end}}
+          {{if .Values.enable_metrics}}- --enable_metrics={{ join "," .Values.enable_metrics}}{{end}}
+          {{if .Values.port}}- --port={{.Values.port}}{{end}}
+          {{if .Values.docker_only}}- --docker_only={{.Values.docker_only}}{{end}}
+        securityContext:
+          {{- toYaml .Values.securityContext | nindent 10 }}
+        resources:
+          requests:
+            memory: 400Mi
```

Comment:
why did we pick these values?

Reply:
Source:  https://github.com/google/cadvisor/blob/master/deploy/kubernetes/base/daemonset.yaml#L23